src/card.css

```diff
@@ -0,0 +1,124 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 166px;
+
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image-wrap {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  padding: 0 3px;
+  padding-bottom: 40px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__title {
+  margin: 0;
```

Comment:
You can create a separate reset.css file for all reset actions

Reply:
I know, but the task says we must have only two blocks, the card, and stars.